framework/src/materials/PiecewiseLinearInterpolationMaterial.C

```diff
@@ -0,0 +1,87 @@
+/****************************************************************/
```

Comment:
This is fine. If others are in favor of dropping generic from the existing similar objects. I'm also fine with that.